releases/v3.4.0/developers/obo_nodes/math_equation.md

```diff
@@ -14,6 +14,7 @@ A LaTeX math equation rendered with [Katex](https://github.com/Khan/KaTeX)
 | Property | Required | Type | Description |
 |-
 | latex | Required | String | The LaTeX string to render (example: `y=\sin(2x)`)
+| alt | Recommended | String | Used as the text content which is read when this equation is encountered by a screen-reader. If not specified the `latex` value is used instead.
```

Comment:
The text to be read when the equation is encountered by a screen-reader. If not specified, the `latex` value is used. It is recommended to include a human-readable form of the equation, as the `latex` value may be difficult for screen-reader users to understand.

Reply:
Updated